module/apmgin/middleware.go

```diff
@@ -18,10 +18,8 @@
 package apmgin // import "go.elastic.co/apm/module/apmgin"
 
 import (
-	"net/http"
-	"sync"
-
 	"github.com/gin-gonic/gin"
+	"net/http"
```

Comment:
please move this back to the top in its own group

`make fmt` should fix it

Reply:
The import groups are still wrong - this is causing `make check` to fail in CI: https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-go%2Fapm-agent-go-mbp/detail/PR-983/5/pipeline/134

Please add an empty line after "github.com/gin-gonic/gin".